portfolio/src/main/webapp/comments.js

```diff
@@ -59,6 +59,34 @@ class Comments {
     return commentElement;
   }
 
+  /**
+   * Deletes all comments in DataStore by doing a POST to /delete-comments-data.
+   * Then, the comments are reloaded to confirm the database has been cleared.
+   */
+  deleteComments() {
+    const request = new Request('/delete-comments-data', {method:'POST'});
+    fetch(request)
+    .then(this.checkPostError)
+    .then(this.getComments(1))
+    .then(this.checkDeleteError)
```

Comment:
Not needed. Your deleteComments request should return an error if it didn't actually delete all the comments.